optima/project.py

```diff
@@ -382,9 +383,69 @@ def runbudget(self, budget=None, budgetyears=None, progsetname=None, parsetname=
     
     def minoutcomes(self, name=None, parsetname=None, progsetname=None, inds=0, objectives=None, constraints=None, maxiters=1000, maxtime=None, verbose=5, stoppingfunc=None, method='asd'):
         ''' Function to minimize outcomes '''
-        optim = Optim(project=self, name=name, objectives=objectives, constraints=constraints, parsetname=parsetname, progsetname=progsetname)
+        optim = Optim(project=self, name=name, which='outcome', objectives=objectives, constraints=constraints, parsetname=parsetname, progsetname=progsetname)
         multires = minoutcomes(project=self, optim=optim, inds=inds, maxiters=maxiters, maxtime=maxtime, verbose=verbose, stoppingfunc=stoppingfunc, method=method)
         self.addoptim(optim=optim)
         self.addresult(result=multires)
         self.modified = today()
         return None
+    
+    
+    def minmoney(self, name=None, parsetname=None, progsetname=None, inds=0, objectives=None, constraints=None, maxiters=200, maxtime=None, verbose=5, stoppingfunc=None, debug=False):
+        ''' Function to minimize money '''
+        optim = Optim(project=self, name=name, which='money', objectives=objectives, constraints=constraints, parsetname=parsetname, progsetname=progsetname)
+        multires = minmoney(project=self, optim=optim, inds=inds, maxiters=maxiters, maxtime=maxtime, verbose=verbose, stoppingfunc=stoppingfunc, debug=debug)
+        self.addoptim(optim=optim)
+        self.addresult(result=multires)
+        self.modified = today()
+        return None
+        
+        
+    #######################################################################################################
+    ## Methods to handle specialised tasks (i.e. for geospatial analysis)
+    #######################################################################################################
+        
+    def genBOC(self, budgetlist=[10000,100000,1000000,10000000], name=None, parsetname=None, progsetname=None, inds=0, objectives=None, constraints=None, maxiters=1000, maxtime=None, verbose=5, stoppingfunc=None, method='asd'):
```

Comment:
Shouldn't really pass in a list like this, I think. Also, from what I can see, this is a list of the total budgets that the individual projects will be allocated (in the loop below), is that right? If so, wouldn't we (1) want higher numbers (e.g. Zambia's budget is ~$500m) and/or (2) want to use a scaling factor of the actual total budget rather than setting an amount?